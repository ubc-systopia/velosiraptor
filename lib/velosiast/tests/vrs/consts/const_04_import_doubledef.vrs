import const_00_defs;

const FOO1: int = 1234;
const FOO2: int = 0xabcd;
const BAR3: int = FOO1 + FOO2;