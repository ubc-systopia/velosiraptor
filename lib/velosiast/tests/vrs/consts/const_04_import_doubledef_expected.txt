error: duplicate definitions with name `FOO1`
     --> tests/vrs/consts/const_04_import_doubledef.vrs:3:1
      |
    1 |  const FOO1 : int = 123;
      |  ^^^^^^^^^^^^^^^^^^^^^^^ previous definition of `FOO1` was here
   ...
      |
    3 |  const FOO1: int = 1234;
      |  ^^^^^^^^^^^^^^^^^^^^^^^ duplicate definition
error: duplicate definitions with name `FOO2`
     --> tests/vrs/consts/const_04_import_doubledef.vrs:4:1
      |
    2 |  const FOO2 : int = 0x1234;
      |  ^^^^^^^^^^^^^^^^^^^^^^^^^^ previous definition of `FOO2` was here
   ...
      |
    4 |  const FOO2: int = 0xabcd;
      |  ^^^^^^^^^^^^^^^^^^^^^^^^^ duplicate definition
error: could not compile due to 2 previous errors; 0 warnings emitted
